### PR TITLE
Contributing

### PR DESCRIPTION
**Pull request by jhkennedy**
Add contributing guidelines to support external contributions better

Also, catch-up the changelog since we ask contributors to update it in `CONTRIBUTING.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+Thank you for considering to contribute and making custom on-demand SAR processing accessible!
+
+We're excited you would like to contribute to HyP3! Whether you're finding bugs, adding new features, fixing anything broken, or improving documentation, get started by submitting an issue or pull request!
+
+## Submitting an Issue
+
+If you have any questions or ideas, or notice any problems or bugs, first [search open issues](https://github.com/ASFHyP3/hyp3-lib) to see if the issue has already been submitted. We may already be working on the issue. If you think your issue is new, you're welcome to [create a new issue](https://github.com/ASFHyP3/hyp3-lib/issues/new).
```
Comment (Jlrine2) [inline CONTRIBUTING.md:9]: shouldn't this link to the issues tab?

Comment (jhkennedy) [reply CONTRIBUTING.md:9]: Yes, yes it should.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+   ![pr base selection](docs/imgs/fork-pr-bases.png?raw=true)
+
+## Guidelines
+
+We ask that you follow these guidelines with your contributions
+
+### Style
+
+We generally follow python community standards ([PEP8](https://pep8.org/)), expect we allow line lengths up to 120 characters. We recommend trying to keep lines 80--100 characters long, but allow up to 120 when it improves readability.
```
Comment (khogenso) [inline CONTRIBUTING.md:61]: "except"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+Thank you for considering to contribute and making custom on-demand SAR processing accessible!
```
Comment (khogenso) [inline CONTRIBUTING.md:3]: Wording feels awkward, maybe: Thank for you considering helping make custom on-demand SAR processing accessible!

File: CONTRIBUTING.md (hunk 2 above)
Comment (jhkennedy) [reply CONTRIBUTING.md:61]: Woops!

File: CONTRIBUTING.md (hunk 3 above)
Comment (jhkennedy) [reply CONTRIBUTING.md]: You're right. Hmm... maybe:

> Thank you for your interest in helping make custom on-demand SAR processing accessible!

Comment (khogenso) [reply CONTRIBUTING.md]: Perfect!

**Pull request merged**